apps/bbsync/tests/test_integration.py

```diff
@@ -444,7 +444,12 @@ def test_flaw_unembargo_complex(
         )
         ps_module1 = PsModuleFactory(
             bts_name="bugzilla",
-            bts_groups={"public": ["devel"]},
+            bts_groups={
+                "public": ["devel"],
+                "embargoed": [
+                    "private",
```

Comment:
Why `"embargoed": ["private"]` is necessary now?

Reply:
An "embargoed" group seems to be necessary when a flaw is embargoed and synced with bugzilla, as per this line which otherwise causes a `KeyError` https://github.com/RedHatProductSecurity/osidb/blob/90f7256b55a60606a134a157cae50835bf8e1760/apps/trackers/bugzilla/query.py#L221

This test started failing with this commit's modification, probably because this inconsistency wasn't caught before due to the same error solved by it.